cpp/perspective/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "@finos/perspective-cpp",
+    "private": true,
+    "author": "The Perspective Authors",
+    "license": "Apache-2.0",
+    "version": "0.6.0",
+    "main": "./dist/esm/perspective.cpp.js",
+    "files": [
+        "dist/esm/**/*",
+        "dist/cjs/**/*"
+    ],
+    "scripts": {
+        "build": "yarn emsdk-run node ./build.js",
+        "clean": "rimraf dist/cpp/build",
+        "test": ":",
+        "test:run": ":",
+        "test:build": ":"
+    }
+}
```

Comment:
`@finos/perspective-cpp` packages adds this build cycle to the build yarn workspaces build graph, and wraps up the minimal leftover JavaScript build scripts.  Even though this is private, we use `"files"` param because we will later on use `npm-pack-here` to "deploy" this module within `@finos/perspective/dist`, effectively static-linking modules in JavaScript 💯